### PR TITLE
Docs cleanup

### PR DESCRIPTION
**Pull request by mortend**
Some clean-ups in preparation for the upcoming NPM release.

Because `README.md` will be added to Uno's package page on npmjs.com, build instructions are extracted to a separate document. I will add install instructions to `README.md` and update `CHANGELOG.md` on the upcoming release-branch, which should be created shortly after this and #150 are merged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,61 +73,12 @@ The following foreign languages are supported:
...
-
-On Windows, some additional runtimes might be needed.
-* VCRedist 2010: [x86](https://www.microsoft.com/en-us/download/details.aspx?id=5555), [x64](https://www.microsoft.com/en-US/Download/confirmation.aspx?id=14632)
-* [VCRedist 2013](https://www.microsoft.com/en-gb/download/details.aspx?id=40784)
-
 ## Contributing
 
-Please read [CONTRIBUTING.md](CONTRIBUTING.md) for details on our code of
+Please read [build-instructions](docs/build-instructions.md) for details
+on how to build the source code.
```
Comment (kusma) [inline README.md]: I don't think this should be nested under "## Contributing"...

Comment (mortend) [reply README.md]: I have moved this paragraph to its own heading now.

File: README.md [hunk 2]
```diff
@@ -67,67 +73,20 @@ The following foreign languages are supported:
...
 
 ### Reporting issues
 
 Please report issues [here](https://github.com/fuse-open/uno/issues).
 
+## Build instructions
+
+Please read [the build instructions](docs/build-instructions.md) for details
+on how to build the source code.
+
```
Comment (kusma) [inline README.md]: Structually, I would think it's better to put this where the old "## Building"-header was. It's kinda more important information than contributing, I think...

Comment (mortend) [reply README.md]: I agree and have moved the section.

**Pull request merged**